puppeteer-scraper/INPUT_SCHEMA.json

```diff
@@ -168,17 +188,15 @@
             "type": "integer",
             "description": "Maximum time the scraper will allow a web page to load in seconds.",
             "minimum": 1,
-            "default": 60,
-            "maximum": 360,
+            "default": 86400,
```

Comment:
I would be afraid to increase these timeouts as many anti-scraping protections let client timeout which would cause this to run too long.

Reply:
I agree. 86400 is high. I will revert back to original values.

I had a timeout problem and thought these setting was the cause. the real cause ended up being the actor timeout setting.